components/Navigation/navigation.js

```diff
@@ -0,0 +1,26 @@
+const menuIcon = document.querySelector('.menu-btn');
+menuIcon.addEventListener('click', menuHandler);
+const sideBar = document.querySelector('.side-overlay');
+function menuHandler() {
+    console.log('clcikd')
+    sideBar.classList.add('open');
```

Comment:
Remove `console.log` if it's not needed.

Reply:
Removed